OIPA/api/transaction/serializers.py

```diff
@@ -229,7 +231,7 @@ def validate(self, data):
             # data.get('sector', {}).get('code'),
             # data.get('sector', {}).get('vocabulary'),
             # data.get('sector', {}).get('vocabulary_uri'),
-            data.get('recipient_country', {}).get('code'),
```

Comment:
Also, please update the tests in here https://github.com/zimmerman-zimmerman/OIPA/blob/develop/OIPA/api/activity/tests/test_save_serializers.py to test your changes